07-documenting_R.Rmd

```diff
@@ -98,10 +96,14 @@ Our dev guide workflow uses continuous integration and deployment so changes to
 
 ## Instructions for reporting the CRAN policy bugs -- discussion in slack (random channel)
 
+If you find some typo or problem on [CRAN](https://cran.r-project.org) after checking the problem or typo you can write a polite email to `cran-sysadmin@r-project.org` and one of the R-core members working with CRAN. You'll probably get a prompt reply about how the issue is going to be fixed.
+
 **Note**:
 
 There is a `#core-documentation` channel on the [R Contributors slack](https://r-contributors.slack.com/) where you can discuss about the patches for improvements to R's documentation. 
 
 ## See also
 
 1. [Writing R documentation files](https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Writing-R-documentation-files)
+2. [R manuals](https://cran.r-project.org/manuals.html): These are a part of the [R sources](https://github.com/r-devel/r-svn/tree/master/doc/manual). Hence, bug reports/patches can also be submitted via Bugzilla, e.g. https://bugs.r-project.org/bugzilla/show_bug.cgi?id=15221. Note that they are typically referred to by their file names "R-admin", "R-exts" etc. The manuals are written in [Texinfo](https://www.gnu.org/software/texinfo/) (file extension `.texi`). The manual should be referred to for [how to mark up text](https://www.gnu.org/software/texinfo/manual/texinfo/texinfo.html). 
```

Comment:
This part should be elevated to a new section. Since you already cite R-exts earlier, you can probably drop this See also section.

To make this guide more self-contained, you should briefly describe the bug you link to and use a hyperlink with a title of the linked bug: [Bug 15221 - R-admin/'Installing R under Windows': Missing argument name](https://bugs.r-project.org/show_bug.cgi?id=15221).

Also, you could add a list or table with the manual names and their nicknames.

"The [Texinfo manual](https://www.gnu.org/software/texinfo/manual/texinfo/texinfo.html) should be referred to for [how to mark up text]."